packages/openneuro-app/src/index.html

```diff
@@ -2,6 +2,13 @@
 <html>
 
 <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date())
+    </script>
```

Comment:
Looks like this won't work because of the id defined here. Can we just include the script tag here with src="https://www.googletagmanager.com/gtag/js" and do any initialization in client.jsx?